homeassistant/components/iron_os/update.py

```diff
@@ -92,4 +92,4 @@ async def async_added_to_hass(self) -> None:
     @property
     def available(self) -> bool:
         """Return if entity is available."""
-        return super().available and self.firmware_update.last_update_success
+        return bool(self.installed_version) and self.firmware_update.last_update_success
```

Comment:
Wait what are we checking here?

Reply:
Checking that we actually have a firmware version for the device in memory. If the firmware version hasn't been fetched from the device yet, installed_version will be None and the entity will also stay unavailable.

Reply:
but why do we use `bool`? Can't we use `is None`?

Reply:
Oh, that makes totally sense. Sorry, my thought processes are sometimes a bit weird 😬